types/json-schema/index.d.ts

```diff
@@ -22,12 +22,18 @@ export type JSONSchema4Type = any[] | boolean | number | null | object | string
 
 /**
  * Meta schema
+ *
+ * Recommended values:
+ * - 'http://json-schema.org/schema#'
+ * - 'http://json-schema.org/hyper-schema#'
+ * - 'http://json-schema.org/draft-04/schema#'
+ * - 'http://json-schema.org/draft-04/hyper-schema#'
+ * - 'http://json-schema.org/draft-03/schema#'
+ * - 'http://json-schema.org/draft-03/hyper-schema#'
+ *
  * @see https://tools.ietf.org/html/draft-handrews-json-schema-validation-01#section-5
  */
-export type JSONSchema4Version = 'http://json-schema.org/schema#' | 'http://json-schema.org/hyper-schema#'
-    | 'http://json-schema.org/draft-04/schema#' | 'http://json-schema.org/draft-04/hyper-schema#'
-    | 'http://json-schema.org/draft-03/schema#' | 'http://json-schema.org/draft-03/hyper-schema#'
-    | string;
+export type JSONSchema4Version = string;
```

Comment:
No need to have `XVersion` types then.

Reply:
We can’t remove it, since it’s exported. Removing it could break existing code.

Reply:
Right, sorry, I've misread the Git diff and thought your PR added them.

Reply:
No you're right, I did add them, but only because `JSONSchema7Version` already existed. I just wanted them to be more consistent. We *could* remove `JSONSchema4Version` and `JSONSchema6Version`, but we cannot remove `JSONSchema7Version`, and I figured consistency was more important than optimization in this case.